pom.xml

```diff
@@ -223,7 +222,7 @@
                 <artifactId>maven-assembly-plugin</artifactId>
                 <version>2.4</version>
                 <configuration>
-                    <finalName>${project.artifactId}-${project.customVersion}</finalName>
```

Comment:
I think this might break rendang at the moment, is there a way to pass a dummy custom version on the docker build so it solely builds `/target/json-wikipedia-jar-with-dependencies.jar` ?

Reply:
maybe doable like: `maven ... -Dproject.customVersion=my_default_v`